src/core/textures/Texture.js

```diff
@@ -40,8 +40,9 @@ export default class Texture extends EventEmitter
      * @param {PIXI.Rectangle} [orig] - The area of original texture
      * @param {PIXI.Rectangle} [trim] - Trimmed rectangle of original texture
      * @param {number} [rotate] - indicates how the texture was rotated by texture packer. See {@link PIXI.GroupD8}
+     * @param {PIXI.Point} [anchor] - Default anchor point used for sprite placement / rotation
      */
-    constructor(baseTexture, frame, orig, trim, rotate)
+    constructor(baseTexture, frame, orig, trim, rotate, anchor)
```

Comment:
If a default argument value is set here: `anchor = new Point(0, 0)`, you don't need to check for it later. Little cleaner.

Reply:
that's extra object allocation because it assigns "anchor" later too.

Reply:
I don't feel strongly about this.